zaga-web/src/app/components/creatework-order/creatework-order.component.html

```diff
@@ -1,857 +1,1708 @@
+                    >
+                      <i class="fa fa-check"></i>
+                    </button>
+                  </div>
+                  <div *ngIf="editing" class="form-group">
+                    <button
+                      class="btn btn-primary btn-add mr-2"
+                      (click)="editExistingWorker()"
+                      [disabled]="workOrder.status == 'Zatvoren'"
+                    >
+                      <i class="fa fa-check"></i>
+                    </button>
+                  </div>
+                </td>
+              </tr>
+              <tr *ngFor="let worker of employees">
+                <td>{{ worker.name }}</td>
+                <td *ngIf="!new">{{ worker.date }}</td>
+                <td>{{ worker.operation }}</td>
+                <td *ngIf="!new">{{ worker.dayWorkPeriod || 0 }} h</td>
+                <td *ngIf="!new">{{ worker.nightWorkPeriod || 0 }} h</td>
+                <td *ngIf="!new">
+                  {{ worker.dayWorkPeriod * 1 + worker.nightWorkPeriod * 1 }} h
+                </td>
+                <td>
+                  <button
+                    class="btn btn-primary btn-edit mr-2"
+                    (click)="editWorker(worker)"
+                    [disabled]="workOrder.status == 'Zatvoren'"
+                  >
+                    <i class="fa fa-edit"></i>
                   </button>
-                </div>
-              </td>
-            </tr>
-            <tr *ngFor="let worker of employees">
-              <td>{{ worker.name }}</td>
-              <td *ngIf="!new">{{ worker.date }}</td>
-              <td>{{ worker.operation }}</td>
-              <td *ngIf="!new">{{ worker.dayWorkPeriod || 0}} h</td>
-              <td *ngIf="!new">{{ worker.nightWorkPeriod || 0}} h</td>
-              <td *ngIf="!new">{{ worker.dayWorkPeriod*1 + worker.nightWorkPeriod*1 }} h</td>
-              <td>
-                <button class="btn btn-primary btn-edit" (click)="editWorker(worker)"
-                  [disabled]="workOrder.status == 'Zatvoren'">
-                  <i class="fa fa-edit"></i>
-                </button>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </form>
       </div>
     </div>
-
-    <!--machines-->
-    <div class="machines mt-4">
-      <button class="btn btn-primary collapsing-button shadow-none" (click)="expandMachines()" type="button"
-        data-toggle="collapse" data-target="#collapseExampleMachines" aria-expanded="false"
-        aria-controls="collapseExampleMachines">
-        <i class="fa fa-truck mr-2"></i>Mašine<i *ngIf="!machines" class="fa fa-arrow-down mr-2 mt-1 arrows"></i><i
-          *ngIf="machines" class="fa fa-arrow-up mr-2 mt-1 arrows"></i>
-      </button>
-      <div class="collapse" id="collapseExampleMachines">
-        <div class="card card-body collapse-card">
+  </div>
+  <!--machines-->
```

Comment:
After adding the machine screenshot

![image](https://user-images.githubusercontent.com/30258800/94562646-df863880-0265-11eb-9084-563ae4d90671.png)